test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoUpdateSuite.java

```diff
@@ -176,77 +166,6 @@ private HapiApiSpec usdFeeAsExpected() {
 				);
 	}
 
-	private HapiApiSpec updateFailsWithInvalidMaxAutoAssociations() {
-		final int tokenAssociations_restrictedNetwork = 10;
-		final int tokenAssociations_adventurousNetwork = 1_000;
```

Comment:
Is it intentionally removed ? Or should this test be modified as per the current change

Reply:
this restriction is no longer valid. Intentionally removed.